product_docs/docs/pem/9/tuning_performance/performance_diagnostic.mdx

```diff
@@ -77,35 +78,57 @@ By default, the top Performance Diagnostic graph pulls the data of the last hour
 
 To open the Performance Diagnostic dashboard in a new browser tab, select **Preferences > Open in New Browser Tab?**.
```

Comment:
GitHub won't let me comment on this sentence (line 77) because it's not part of the diff, but this doesn't make sense to me. What is the 15 seconds about? I think the sentence would be correct if we just removed "and in the last 15 seconds", what do you think @asheshv ?

> Each point of this time series represents the active sessions along with the wait events at a particular time and in the last 15 seconds.

Reply:
@sonotley - Each point is sum of the wait events & CPU in last 15 seconds from that moment.
Actually - it does not represent number of active sessions for sure.

We can change it to something like as below:
> Each point of this time series represents the sum of wait events & CPU (aggregated by 15 seconds).

Reply:
@sonotley, let me know your thoughts on this.

Reply:
I didn't know we were performing aggregation, I thought each point was a single sample - how else does the sample selection behaviour work @asheshv ? If we are aggregating then I suggest: 

> Each point of this time series represents the sum of wait events & CPU over the previous 15 seconds.